### PR TITLE
Fixed DownloadPackageV0 task to be able to download AFD enabled packages

### PR DESCRIPTION
**Pull request by aasim**

### CODE REVIEW

File: Tasks/DownloadPackageV0/connections.ts [hunk 1]
```diff
@@ -5,39 +5,22 @@ import { IRequestOptions } from "azure-devops-node-api/interfaces/common/VsoBase
...
-            options.headers["X-TFS-FedAuthRedirect"] = "Suppress";
-        }
-    }
-
-    isPreSignedUrl(options: any): boolean {
-        return (
-            options.host &&
-            options.host.endsWith("blob.core.windows.net") &&
-            options.path &&
-            options.path.includes("&sig=")
```
Comment (shubham90) [inline Tasks/DownloadPackageV0/connections.ts]: why are we removing this check?

File: Tasks/DownloadPackageV0/task.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "version": {
         "Major": 0,
         "Minor": 161,
-        "Patch": 0
```
Comment (shubham90) [inline Tasks/DownloadPackageV0/task.json]: I though you already made this 161.2? shouldn't this be more than that?

Comment (aasim) [reply Tasks/DownloadPackageV0/task.json]: This is DownloadPackageV0. The 161.2 was for DownloadPackageV1

File: Tasks/DownloadPackageV0/connections.ts (hunk 1 above)
Comment (aasim) [reply Tasks/DownloadPackageV0/connections.ts]: I used the updated library for this task that uses Regex to match the urls. The regex using just the hostname works as well, and its too complicated with additional options.

**Pull request merged**